src/test/scala/io/iohk/ethereum/network/handshaker/EtcHandshakerSpec.scala

```diff
@@ -162,7 +162,7 @@ class EtcHandshakerSpec extends FlatSpec with Matchers  {
         override val blockExtraData: Option[ByteString] = None
         override val range: Int = 10
         override val drainList: Seq[Address] = Nil
-        override val forkBlockHash: ByteString = ByteString(Hex.decode("94365e3a8c0b35089c1d1195081fe7489b528a84b22199c916180db8b28ade7f"))
+        override val forkBlockHash: ByteString = ByteString(Hex.decode("4afa298dfbb357f502445e3e91367b4bb4faecd1ac0b2a6c09550d18c5f9e429"))
```

Comment:
Maybe we should replace this with `DaoForkConfig(Config.config).forkBlockHash` so that we don't forget to have to change this hash?

Reply:
:+1:

Reply:
Its weird, I can't replace to `DaoForkConfig(Config.config).forkBlockHash` because it fail to return the value from the config :S I have no clue why..